app/src/main/java/org/kiwix/kiwixmobile/settings/SliderPreference.java

```diff
@@ -171,8 +169,10 @@ protected void onBindView(View view) {
   }
 
   private void setGrayState(View view) {
-    boolean enabled = getPreferenceManager().getSharedPreferences().getBoolean(PREF_ZOOM_ENABLED, false);
-    boolean Nightmode = getPreferenceManager().getSharedPreferences().getBoolean(PREF_NIGHTMODE, false);
+    SharedPreferenceUtil sharedPreferenceUtil = new SharedPreferenceUtil(getContext());
+    boolean enabled = sharedPreferenceUtil.getPrefZoomEnabled();
+    boolean Nightmode = sharedPreferenceUtil.nightMode();
```

Comment:
The variable name should begin with a lowercase. If you want you can clean up this file by removing the Hungarian notations wherever used and the lint warnings.